codersdk/provisionerdaemons.go

```diff
@@ -75,6 +83,7 @@ type ProvisionerJob struct {
 	CompletedAt *time.Time           `json:"completed_at,omitempty" format:"date-time"`
 	CanceledAt  *time.Time           `json:"canceled_at,omitempty" format:"date-time"`
 	Error       string               `json:"error,omitempty"`
+	ErrorCode   JobErrorCode         `json:"error_code,omitempty" enums:"MISSING_TEMPLATE_PARAMETER,REQUIRED_TEMPLATE_VARIABLES"`
```

Comment:
Question: Is the `enums:` still required for code gen even after the update of `swaggo`? In my experience it has been able to auto-detect but it might be very specific.

Reply:
I checked it now - unfortunately, swaggo removed enum parts from the docs and `swagger.json`, so I will leave it as is.